apps/web/src/hooks/useUserCakeLockStatus.ts

```diff
@@ -9,13 +9,20 @@ export const useUserCakeLockStatus = () => {
   const { chainId } = useActiveChainId()
   const cakeVaultContract = useCakeVaultContract()
 
-  const { data: userCakeLockStatus = null } = useSWRImmutable(
-    account && chainId === ChainId.BSC ? ['userCakeLockStatus', account] : null,
+  const { data: userCakeLockStatus = null } = useQuery(
+    ['userCakeLockStatus', account],
     async () => {
+      if (!account) return undefined
```

Comment:
Why we still need check `account` in here?
`enabled: Boolean(account && chainId === ChainId.BSC)` will confirm have account and chainId right